src/components/tasks/card/index.tsx

```diff
@@ -36,16 +36,15 @@ import { useUpdateTaskMutation } from '@/app/services/tasksApi';
 import SuggestionBox from '../SuggestionBox/SuggestionBox';
 import { userDataType } from '@/interfaces/user.type';
 import { GithubInfo } from '@/interfaces/suggestionBox.type';
-import userData from '@/helperFunctions/getUser';
+
+let timer: NodeJS.Timeout;
 
 type Props = {
     content: task;
     shouldEdit: boolean;
     onContentChange?: (changeId: string, changeObject: object) => void;
 };
 
```

Comment:
export this type from suggestionBox.type.ts and use it here

Reply:
This is not my component, I have exported all the props of the component that i have created. @Pratiyushkumar